t/transaction.t

```diff
@@ -7,6 +7,15 @@ subtest "params tests" => sub {
   should_throw("ArgumentError", sub { Net::Braintree::Transaction->sale({"foo" => "Bar"}); });
 };
 
+subtest "refund params test" => sub {
```

Comment:
Could we add more thorough tests here? For example, can we test passing different hashes into `refund`?